Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -80,6 +80,7 @@ public class ControllerPopupWindow : EditorWindow
         private static Vector2 horizontalScrollPosition;
 
         private SerializedProperty currentInteractionList;
+        private static List<string> mappedControllerList;
```

Comment:
Should this be `static`? It feels like if one window passes `mappedControllers` to the `Show` method, then the next one doesn't, the null check in `Show` will leave this set to whatever the last window set it to, which could be incorrect data.

I'm not sure if that's a scenario the code can hit at all, but it might be worth making this non-static and passing it via the `window.mappedControllerList` reference like the `Show` method does for things like `currentControllerMapping`